### PR TITLE
Interpret stack trace upon timeout expiration

### PR DESCRIPTION
**Pull request by waltdisgrace**
Resolves: #336.

### CODE REVIEW

File: src/stratis_cli/_error_reporting.py [hunk 1]
```diff
@@ -145,7 +145,13 @@ def interpret_errors(errors):
             and error.get_dbus_name() == "org.freedesktop.DBus.Error.NameHasNoOwner"
         ):
             return "Most likely stratis is unable to connect to the stratisd D-Bus service."
-
+        if (
+            # pylint: disable=bad-continuation
+            isinstance(error, dbus.exceptions.DBusException)
+            and error.get_dbus_name() == "org.freedesktop.DBus.Error.NoReply"
+        ):  # pragma: no cover
+            return """Stratis attempted communication with the daemon, stratisd,
```
Comment (mulkieran) [inline src/stratis_cli/_error_reporting.py]: Please use the approach for multi-line strings that is used at line 120 in this file.

Comment (mulkieran) [inline src/stratis_cli/_error_reporting.py]: Use lowercase for Stratis even though it starts the sentence. We want to be clear that we are talking about the CLI tool, not the project.

File: src/stratis_cli/_error_reporting.py [hunk 2]
```diff
@@ -145,7 +145,13 @@ def interpret_errors(errors):
...
+            over the D-Bus, but stratisd did not respond in the allowed time."""
```
Comment (mulkieran) [inline src/stratis_cli/_error_reporting.py]: Prefer an empty line in between this and the next section.

**Comment by waltdisgrace** [comment]
rebased.

File: src/stratis_cli/_error_reporting.py [hunk 3]
```diff
@@ -145,6 +145,16 @@ def interpret_errors(errors):
             and error.get_dbus_name() == "org.freedesktop.DBus.Error.NameHasNoOwner"
         ):
             return "Most likely stratis is unable to connect to the stratisd D-Bus service."
+        if (
```
Comment (mulkieran) [inline src/stratis_cli/_error_reporting.py]: Please go ahead and add the three line FIXME, like with the other parts of this method that we are not testing.

**Pull request merged**